Deployment/terraform_run.ps1

```diff
@@ -0,0 +1,6 @@
+& cls
+& terraform init -backend-config="resource_group_name=FileShareSerDevP4052-RG" -backend-config="storage_account_name=fileshareserdevp4052" -backend-config="key=terraform.deployment.tfplan"
```

Comment:
FileShare? 😆

Reply:
Backend details are still not available should have kept blank. removed it and kept it blank now.